From 14a13b7561541d2704f692944dc6a75f98817f85 Mon Sep 17 00:00:00 2001
From: sbruno <sbruno@FreeBSD.org>
Date: Sun, 23 Sep 2012 20:16:17 +0000
Subject: [PATCH 6/8] MFC r238373

sys/dev/mfivar.h contains references to MFI_DEBUG, but it never gets turned on
unless the file also includes opt_mfi.h.

Submitted by: Andrew Boyer aboyer@averesystems.com
---
 sys/dev/mfi/mfivar.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sys/dev/mfi/mfivar.h b/sys/dev/mfi/mfivar.h
index f63b64c..e4345cc 100644
--- a/sys/dev/mfi/mfivar.h
+++ b/sys/dev/mfi/mfivar.h
@@ -61,6 +61,7 @@ __FBSDID("$FreeBSD$");
 
 #include <sys/types.h>
 #include <sys/taskqueue.h>
+#include "opt_mfi.h"
 
 /*
  * SCSI structures and definitions are used from here, but no linking
-- 
1.7.9.5

